MvvmCross/Platform/Wpf/Views/MvxApplication.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Threading;
+using MvvmCross.Core;
+using MvvmCross.Platform.Wpf.Core;
+using MvvmCross.ViewModels;
+
+namespace MvvmCross.Platform.Wpf.Views
+{
+    public abstract class MvxApplication : Application
+    {
+        private MvxWpfSetup _setup;
+        protected MvxWpfSetup Setup
+        {
+            get
+            {
+                if (_setup == null)
+                    _setup = CreateSetup(Dispatcher, MainWindow);
```

Comment:
Can we make WPF work the same way as Android and iOS? (use a `InitializePlatform` method instead of ctor arguments) It seems cleaner to me to unify all setup changes on the same PR